.github/workflows/regular.yml

```diff
@@ -40,17 +40,20 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - uses: actions/setup-python@v5
-        with: {python-version: "3.9"}
+        with: {python-version: "3.12"}
```

Comment:
Why using 3.12 for the docs instead of 3.9?

Reply:
That change was done by @Scienfitz. To me, it doesn't matter, but maybe he has his reasons? Let's see what he says ...

Reply:
it also appear that the docs building in the CI does not fail, despite py >3.9, this must be due to it not buidling the examples (via -e) which means the examples might be behind the ongoing pycache issues